x-pack/plugins/security_solution/cypress/screens/search_bar.ts

```diff
@@ -28,3 +28,13 @@ export const ADD_FILTER_FORM_FILTER_VALUE_INPUT = '[data-test-subj="filterParams
 export const ADD_FILTER_FORM_SAVE_BUTTON = '[data-test-subj="saveFilter"]';
 
 export const GLOBAL_SEARCH_BAR_FILTER_ITEM = '#popoverFor_filter0';
+
+export const GLOBAL_SEARCH_BAR_SAVE_QUERY =
+  '[data-test-subj="saved-query-management-popover-button"] :not([data-test-subj="timeline"] [data-test-subj="saved-query-management-popover-button"])';
```

Comment:
Trying to avoid selecting the one in the timeline

Reply:
something like [class^="FiltersGlobalContainer"] [data-test-subj="saved-query-management-popover-button"] would be a little less brittle than :not I think